app/models/site.rb

```diff
@@ -7,6 +7,20 @@ class Site
   cattr_accessor :preloaded_category_custom_fields
   self.preloaded_category_custom_fields = Set.new
 
+  ##
```

Comment:
btw, what's this style of `##` comments? 🤔

Reply:
@CvX IDK where exactly I picked that up from, maybe yard? I usually do them to kind of "fence in" the top of big blocks of comments